README.md

```diff
@@ -109,9 +109,26 @@ Server is running.
     EOF
     ```
 
-    The user can also be restricted to only some buckets
-    (foo bucket for example) by changing the `Resource` property to
-    `"Resource": "arn:scality:utapi:::buckets/foo"`
+    A user's access may be limited to certain buckets or the resources of a
+    certain account using the `Resource` property.
+
+    Sample ARN limiting the user's access to metrics of a bucket 'foo':
+
+    ```
+    "Resource": "arn:scality:utapi:::buckets/foo"
+    ```
+
+    Sample ARN limiting the user's access to metrics of an account with the ID
+    '012345678901':
+
+    ```
+    "Resource": "arn:scality:utapi::012345678901:buckets/*"
```

Comment:
Let's change the default policy to include accountId. And here we can refer to on how to access metrics for all accounts.